source/types/globals.d.ts

```diff
@@ -20,7 +20,7 @@ type AnyObject = Record<string, any>;
 declare module 'select-dom' {
 	import 'select-dom';
 
-	function select(selector: string, parent?: any): any | null;
+	function select<T extends HTMLElement = HTMLElement>(selector: string, parent?: string|Element|DocumentFragment): T | null;
```

Comment:
Modified the types here so that we get more information. `HTMLElement` as the default value for `T` is generally fine unless we need specific attributes for a type of HTML Element, e.g.

```
// Type Error: Property 'value' does not exist on type 'HTMLElement'. ts(2339)
select('.some-checkbox-selector').value

// All good
select<HTMLInputElement>('.some-checkbox-selector').value
```

Also, in some cases the parent is a `DocumentFragment`.